dev/docker-compose.yaml

```diff
@@ -21,31 +21,25 @@ services:
     image: confluentinc/cp-kafka:6.2.0
     networks:
       kafka:
+    depends_on:
+    - broker
     command: >
      /bin/bash -c "
-        while ! nc -z broker 9092;
```

Comment:
I added this hack as the `depends` token waits on service "readyness". Defining this term is up to the service and must not mean that the service is ready to accept requests.

If this bash block doesn't cause any issues I'd be inclined to keep it in as it guards against flakyness of tests.

Reply:
I remove this hack as `while` may loop forever if the 9092 port is not ready, especially at local dev environment when I want do some individual code and test.

It's safe to remove the hack as I can see the `healtcheck` in *broker* container.